### ISSUE TITLE
Watch issue when using // comment

### ISSUE THREAD

**Issue by paulduthoit** [elliottsj/babel-inline-import-loader#25]
Hi!

After hours of debugging, i found a bug in the watcher of babel-inline-import-loader.

Here, the watcher works properly :
```js
import React from 'react';
import testString from './test.md';
```

Here, the watcher doesn't work properly :
```js
import React from 'react';

// This comment prevent the watcher to works properly
import testString from './test.md';
```

It's very annoying because the watcher will no longer watch the `test.md` file. Please help us... :(

**Comment by paulduthoit** [comment]
🙏

**Comment by elliottsj** [comment]
Released as v1.0.1

**Comment by paulduthoit** [comment]
Thanks @elliottsj for being so responsive. It works great.
